ui/src/app/workflows/components/workflows-summary-container/workflows-summary-container.tsx

```diff
@@ -18,6 +19,7 @@ export function WorkflowsSummaryContainer(props: {workflows: Workflow[]}) {
             setWfSummary(summary);
         }
     }, [props.workflows]);
+    const {t} = useTranslation();
```

Comment:
same as above, this should be before the `useEffect` with the other variable declarations above